src/auro-select.js

```diff
@@ -244,6 +251,22 @@ class AuroSelect extends LitElement {
       this.shadowRoot.querySelector('auro-dropdown').removeAttribute('disabled');
     }
 
+    this.menus = [...this.querySelectorAll('auro-menu')];
+
+    for (let index = 0; index < this.menus.length; index += 1) {
+      if (this.checkmark) {
+        this.menus[index].removeAttribute('nocheckmark');
+      } else {
+        this.menus[index].setAttribute('nocheckmark', '');
+      }
+    }
+
+    if (this.checkmark) {
+      this.querySelectorAll('auro-menu').forEach((menu) => menu.removeAttribute('nocheckmark'));
+    } else {
+      this.querySelectorAll('auro-menu').forEach((menu) => menu.setAttribute('nocheckmark', ''));
+    }
+
```

Comment:
I believe this code should be removed because we are now doing it in the for loop above at line 254.

Reply:
Duplicate code removed.